src/rules/block-no-empty/__tests__/index.js

```diff
@@ -48,3 +48,17 @@ testRule(rule, {
     column: 18,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
+  skipBasicChecks: true,
+  syntax: "sugarss",
+
+  reject: [{
+    code: ".a",
+    message: messages.rejected,
+    line: 1,
+    column: 3,
```

Comment:
`a` is at the second column: columns are 1-based, unlike `index` in loops.

So it's 
- column 1 = `.`
- column 2 = `a`